### PR TITLE
I like making pull requests

### PR DESCRIPTION
**Pull request by dmoreno-harveyRoad**
Pull requests are nice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # GitHub-Example
 
 This is a line edited with the online editor
+
+This is a line added in a different branch.
```
Comment (dmoreno-harveyRoad) [inline README.md]: You need a newline there.

Comment (dmoreno-harveyRoad) [reply README.md]: Of course not

Comment (dmoreno-harveyRoad) [reply README.md]: You're dumb

Comment (Galithil) [reply README.md]: Yeah I think you don't need a newline there.

**Pull request merged**